File: WowUp.Common/Models/GitHub/GitHubRepository.cs
using Newtonsoft.Json;

namespace WowUp.Common.Models.GitHub
{
    public class GitHubRepository
    {
        public long Id { get; set; }

        [JsonProperty("node_id")]
        public string NodeId { get; set; }

        public string Name { get; set; }

        [JsonProperty("full_name")]
        public string FullName { get; set; }

        public bool Private { get; set; }

        public GitHubUploader Owner { get; set; }

        [JsonProperty("html_url")]
        public string HtmlUrl { get; set; }

        public string Description { get; set; }

        public bool Fork { get; set; }

        public string Url { get; set; }
    }
}

File: wowup-electron/src/app/models/github/github-repository.ts
import { GitHubUploader } from "./github-uploader";

export interface GitHubRepository {
  id: number;
  node_id: string;
  name: string;
  full_name: string;
  private: boolean;
  owner: GitHubUploader;
  html_url: string;
  description: string;
  fork: boolean;
  url: string;
}
